deploy/docker-compose/README.md

```diff
@@ -0,0 +1,404 @@
+    ```
+    ERROR: Named volume "$PWD/.md/mdtm:/var/lib/monai:rw" is used in service "task-manager" but no declaration was found in the volumes section. 
+    ```
+
+    *WHY? The value of `TASK_MANAGER_DATA` is exposed to the Task Manager container as an environment variable in order for Task Manager to map the volume from the host to the MAP container correctly.*
+
+- The Informatics Gateway service is unhealthy and fails to start with the following error if Docker is not up-to-date:
+  ```bash
+  $ docker logs mdl-ig
+  Failed to create CoreCLR, HRESULT: 0x80070008
+  ```
+  
+## Running a MONAI Deploy Workflow
+
+This package includes Orthanc running and connected to the Informatics Gateway, with all required AE Titles pre-configured.
+
+To get started, download & unzip the following DICOM datasets and upload them to Orthanc.
+
+- [Chest CT dataset](https://drive.google.com/file/d/1IGXUgZ7NQCwsix57cdSgr-iYErevqETO/view?usp=sharing)
+- [Abdomen CT dataset](https://drive.google.com/file/d/1d8Scm3q-kHTqr_-KfnXH0rPnCgKld2Iy/view?usp=sharing)
+
+Note: these DICOM datasets were converted to DICOM from Medical Decathlon training and validation images and may be used for the referenced examples later in this document.
+
+### Upload DICOM Datasets
+
+Navigate to Orthanc via http://localhost:8042, and click *Upload* on the top right-hand corner.
+Drag & drop any DICOM files (no folders) to the blue box on the page and then click *Start the upload*.
+
+Or, upload your files to Orthanc using the *storescu* command from [dcmtk](https://dcmtk.org/dcmtk.php.en).
+```
+storescu -v +r +sd -aec ORTHANC localhost 4242 /path/to/my/unzipped/dicom/files/*
+```
+
+Navigate to the home page and click *All studies* to confirm data's been uploaded.
+
+## Sample Workflows
+
+The **sample-workflows** directory includes the following four sample workflow definitions:
+
+- `hello-world.json `: Hello World!
+- `lung-seg.json`: AI Lung Segmentation MAP
+- `liver-seg.json`: AI Liver Segmentation MAP
+- `liver-lung-seg.json`: AI Lung & AI Liver Combo
+
+*Note: in these examples, we will be using `curl` command to register MONAI Deploy workflow definitions with the Workflow Manager. To learn more about `curl`, visit the [curl Man Page](https://curl.se/docs/manpage.html).*
+
+### Hello World
+
+#### Description
+
+This example uses the `alpine` container image instead of a MAP to print all files found in the input directory; this example demonstrates how data received by the Informatics Gateway travel through MONAI Deploy and how the data is made available to the application container.
+
+1. Deploy the workflow definition to MONAI Deploy Workflow Manager:
+   ```
+   $ curl --request POST --header 'Content-Type: application/json'  --data "@sample-workflows/hello-world.json"  http://localhost:5001/workflows
```

Comment:
Please remove $, as well as the output from curl, because when MD file is rendered with the "copy", they get copied too, and causes unnecessary errors, which may also confuse some users.

Reply:
Updated